app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Response
+  def respond_with(entity:, status: 200, serializer: nil)
```

Comment:
You should create serializer class for case when `serializer: nil`(use case: error rendering from contract object). Also you should delete `app/controllers/concerns/.keep`.

Reply:
Added.